<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smoking App</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnHide">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSkipped">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Skipped</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSmoked">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Smoked</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnViewStats">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>View Stats</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="settingCigGroup">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Setting For Cigarrete</string>
    </property>
    <widget class="QLineEdit" name="editPackCount">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCountPack">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cigarretes in a pack :</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCostPack">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cost of a pack (USD) :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editCostPack">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblCaution">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Smoking is dangerous to your health ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnViewSetting">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="settingCanGroup">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Setting For Cannabis</string>
    </property>
    <widget class="QLineEdit" name="editGramCount">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCountGram">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Grams at a time(gram):</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCostGram">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cost per gram (USD) :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="editCostGram">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupType">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioCigarrete">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cigarrete</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioCannabis">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cannabis</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="timeLcd">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="timeCountProgress">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>70</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
